.github/workflows/ci.yaml

```diff
@@ -71,6 +71,8 @@ jobs:
         run: |
           cd spark
           ./build/mvn package -Pkubernetes -DskipTests -Dtest=none
+      - name: Run unit tests
+        run: mvn --batch-mode test
```

Comment:
tests are run as part of `mvn install` above:
https://github.com/armadaproject/armada-spark/actions/runs/13935132530/job/39001326184?pr=13#step:7:115